interface/src/app/node/node.js

```diff
@@ -92,5 +92,223 @@ angular.module('adage.node', [
   }]
 )
 
+.directive('highRangeExp', ['$http', '$log', function($http, $log) {
+  return {
+    templateUrl: 'node/high_range_exp.tpl.html',
```

Comment:
Hi @dongbohu: Is this template file for this directive included in the pull request? I can't seem to find it.